test/middleware.test.js

```diff
@@ -39,6 +40,32 @@ describe(`middleware: ${printPath("[test/middleware.test.js]")}`, function () {
         await cleanST();
     });
 
+    //  * TODO: check that disabling default API actually disables it (for session)
+
+    it("test disabling default API actually disables it", async function () {
+        await startST();
+        SuperTokens.init({
+            supertokens: {
+                connectionURI: "http://localhost:8080",
+            },
+            appInfo: {
+                apiDomain: "api.supertokens.io",
+                appName: "SuperTokens",
+                websiteDomain: "supertokens.io",
+            },
+            recipeList: [
+                Session.init({
+                    sessionRefreshFeature: {
+                        disableDefaultImplementation: true,
+                    },
+                }),
+            ],
+        });
+
+        let apisHandled = await SessionRecipe.getInstanceOrThrowError().getAPIsHandled();
+        assert(apisHandled[0].disabled === true);
```

Comment:
No. Create an express app, add the middlewares and error handler, call the refresh API, and check that you get 404